utils/runtime/getting_failure_step.yml

```diff
@@ -0,0 +1,27 @@
+- block:
+
+    ## Getting failure step from experiment-pod
+    - name: Getting failure step from experiment pod
+      shell: >
+        kubectl logs {{ chaos_pod_name }} -n {{ a_ns }} | grep "FAILED!" -B 1 | head -1 | awk -F "Step:" '{print $2}'
```

Comment:
why we don't need `-f` here `kubectl logs -f`?

Reply:
-f is for follow/tail log.. Since we are already in execution flow in this job, we it need to get logs until that point w/ a command that completes.